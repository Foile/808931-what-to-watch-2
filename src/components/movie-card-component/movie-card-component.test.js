import React from 'react';
import renderer from "react-test-renderer";
import MovieCard from './movie-card-component';

it(`App correctly renders after relaunch`, () => {
  const movie = {name: `Macbeth`};
  const tree = renderer
  .create(<MovieCard
    movie = {movie}
    onHeaderClick = {()=>jest.fn()}
    onMouseEnter={()=>jest.fn()} />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});

