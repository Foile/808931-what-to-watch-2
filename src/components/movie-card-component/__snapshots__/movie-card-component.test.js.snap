// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard correctly renders after relaunch 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
  onMouseEnter={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Macbeth"
      height="175"
      src=""
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Macbeth
    </a>
  </h3>
</article>
`;
